examples/qtensor/qtensor.morpho

```diff
@@ -1,17 +1,26 @@
+/* Problem of a 2D nematic confined to a disk with parallel anchoring boundary condition, modeled by a Q-tensor free energy.
+Showcases: AreaIntegral, LineIntegral, MeshRefiner, POVRaytracer, Graphics
+This script showcases a few good aspects of Morpho:
+* It uses AreaIntegral to define the polynomial form of free energy. 
+* It uses mesh refinement to get a better resolution for the solution at the nematic defect locations, where there is more distortion in the field value.
+* To visualize the Q-tensor, we need to see both the director (orientations) and the scalar order parameter (degree of order) together. This script shows how that can be done easily by just adding the graphics together.
+*/
 import meshtools
 import optimize
 import plot
 import povray
 
-var refine_adaptively = false
+var refine_adaptively = false // Flag to turn mesh-refinement on/off
+var visualize_refinement = false // Flag to turn plotting the refinement regions and refined meshes on/off
```

Comment:
Do you want both of these off?

Reply:
I think so. A first time user needn't be bogged down with the long iterations of the refinement. They can just see the result quickly. Then they can change the `refine_adaptively` flag and see the result. To dig deeper in to how the refinement worked, they can turn `visualize_refinement` on. Let me know if that makes sense!

Reply:
Cool, you might note that these are turned off to improve performance but could be turned on to improve accuracy and demonstrate mesh refinement 

(I had assumed mesh refinement would speed up the problem by improving condition)

Reply:
Ah, gotcha. I will add in some explanations. You are right to assume that it would give a speed up, but only in comparison to the case where we would refine uniformly. Here, we are starting with a fairly coarse mesh, and then refining it in only the regions of interest. Perhaps, I'll add in some comment explaining this as well.

Reply:
Gotcha, so the actual use case for this would be to start with a less dense mesh and use refinement to get a faster similar result than with the dense mesh